### PR TITLE
Update docker-run.sh

### PR DESCRIPTION
**Pull request by vipera7**
https://github.com/tuxgasy/docker-dolibarr/issues/27

Update Dolibarr (NOT TESTED !)

### CODE REVIEW

File: docker-run.sh
Comment (mathieupotier) [inline docker-run.sh:131]:
```suggestion
  CURRENT_MAJOR_VERSION=$(echo ${INSTALLED_VERSION} | cut -d. -f1)
```

File: docker-run.sh
Comment (mathieupotier) [inline docker-run.sh:132]:
```suggestion
 NEW_MAJOR_VERSION=$(echo ${TARGET_VERSION} | cut -d. -f1)
```

File: docker-run.sh
Comment (mathieupotier) [inline docker-run.sh:133]:
```suggestion
  for major in `seq ${CURRENT_MAJOR_VERSION} ${NEW_MAJOR_VERSION}`
```

File: docker-run.sh
Comment (mathieupotier) [inline docker-run.sh:135]:
```suggestion
    if [[ "${CURRENT_MAJOR_VERSION}" -eq "${major}" ]]; then
```

File: docker-run.sh
Comment (mathieupotier) [inline docker-run.sh:136]:
```suggestion
      php upgrade.php ${INSTALLED_VERSION} $((major+1)).0.0 >> /var/www/documents/migration_error.html 2>&1 && \
```

File: docker-run.sh
Comment (mathieupotier) [inline docker-run.sh:137]:
```suggestion
      php upgrade2.php ${INSTALLED_VERSION} $((major+1)).0.0 >> /var/www/documents/migration_error.html 2>&1 && \
```

File: docker-run.sh
Comment (mathieupotier) [inline docker-run.sh:138]:
```suggestion
      php step5.php ${INSTALLED_VERSION} $((major+1)).0.0 >> /var/www/documents/migration_error.html 2>&1
```

File: docker-run.sh
Comment (mathieupotier) [inline docker-run.sh:139]:
```suggestion
    elif [[ $((major+1)) != $((NEW_MAJOR_VERSION+1)) ]]; then
```

File: docker-run.sh
Comment (mathieupotier) [inline docker-run.sh:140]:
```suggestion
      php upgrade.php ${major}.0.0 $((major+1)).0.0 >> /var/www/documents/migration_error.html 2>&1 && \
```

File: docker-run.sh
Comment (mathieupotier) [inline docker-run.sh:141]:
```suggestion
      php upgrade2.php ${major}.0.0 $((major+1)).0.0 >> /var/www/documents/migration_error.html 2>&1 && \
```

File: docker-run.sh
Comment (mathieupotier) [inline docker-run.sh:142]:
```suggestion
      php step5.php ${major}.0.0 $((major+1)).0.0 >> /var/www/documents/migration_error.html 2>&1
```

**Review by mathieupotier — CHANGES_REQUESTED** [review]
Some changes suggested in order to stick on CodeStyle in file.

You have 3 cases in your loop to deal with
 - major === CURRENT_MAJOR_VERSION => migrate from CURRENT_MAJOR_VERSION to major +1
 - major === NEW_MAJOR_VERSION => migrate from major to NEW_MAJOR_VERSION
 - and others that migrate from major to major + 1

RC because seems missing the third case IMHO

**Comment by vipera7** [comment]
@mathieupotier Thanks for the reply, I'm sorry but I have no idea how to make changes :( First time doing a Pull requests on github

**Comment by mathieupotier** [comment]
Solution 1 : Just push a commit in your branch `vipera7:patch-1` with the changes I suggested
Solution 2 : you can also click on `commit suggestion` for each messages in this thread, and after pull changes on your branch and chek if everything is OK (commit and push fix if necessary)

Once done, I can re-review and acknowledge it, after @tuxgasy will merge it if he's OK with it.

**Comment by vipera7** [comment]
@mathieupotier Thanks for help. I hope I did it right, here is the updated docker-run.sh : 

https://github.com/vipera7/docker-dolibarr/blob/master/docker-run.sh

**Pull request closed without merging**
